package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commitplease",
-  "version": "2.3.1",
+  "version": "2.4.0",
```

Comment:
Bumping version should only happen as part of `npm version` calls, which also creates a new commit and tags it. Never do that as part of a PR.